package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@krakenjs/belter",
-  "version": "2.6.0",
+  "version": "2.7.1",
```

Comment:
Generally we do not need to update the version. 

But in this PR, github actions already published the branch as `latest` tag on NPM. So the version 2.7.0 already exists from this feature branch. This is to avoid conflict with existing versions.